src/card.py

```diff
@@ -1,3 +1,10 @@
+import json
+#Implementation of a standard Poker card, with ability to output to JSON
 class Card:
-    def __init__(self):
-        print("Card constructor")
+    def __init__(self, rank, suit):
+        self.rank = {"rank": rank}
+        self.suit = {"suit": suit}
+
+    def to_json(self):
+         return json.dumps({"rank": self.rank, 
+                            "suit": self.suit})
```

Comment:
We want to return a `dict` with rank and suit, such as:

```python3
from dataclasses import dataclass

@dataclass
class Card:
    rank: str
    suit: str

    def to_json(self) -> dict[str, str]:
        return {"rank": self.rank, "suit": self.suit}
```

Reply:
Actually, I would disagree here @nfitzen. I think `to_json` suggests the result should be a JSON string (as Olive has done) and not a dict. Otherwise it should be called `to_dict`. :)

That said, the current implementation here will be weird because we're storing both the rank and the suit internally as dicts (as I pointed out above). So the current implementation will return, for example

```
{ "rank": { "rank" : "Q" }, "suit": {"suit": "H" } }
```

Reply:
@semmypurewal this is true, but then I was getting a JSON string within a JSON output. I agree that `to_json` is a poor name and it should be called `to_dict`, but requiring the frontend to deserialize JSON twice is pointless in this case.